src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Forwarders.cs

```diff
@@ -2,14 +2,21 @@
 
 using System.Collections.Immutable;
 using Microsoft.CodeAnalysis.LanguageServices;
-using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.FindSymbols
 {
     internal sealed partial class SyntaxTreeIndex
     {
         public ImmutableArray<DeclaredSymbolInfo> DeclaredSymbolInfos => _declarationInfo.DeclaredSymbolInfos;
 
+        public ImmutableDictionary<string, ImmutableArray<int>> SimpleExtensionMethodInfo
+            => _extensionMethodInfo.SimpleExtensionMethodInfo;
+
+        public ImmutableArray<int> ComplexExtensionMethodInfo
+            => _extensionMethodInfo.ComplexExtensionMethodInfo;
+
+        public bool ContainsExtensionMethod => SimpleExtensionMethodInfo.Count > 0 || ComplexExtensionMethodInfo.Length > 0;
```

Comment:
as these are just forwarders, it would be nice for this to just forward to a `.ContainsExtensionMethod ` on _extensionMethodInfo

Reply:
done